roc/go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/roc-streaming/roc-go
```

Comment:
Well it should be `github.com/roc-streaming/roc-go/roc` then? :)

Reply:
that's right